frontend/src/plugin/registry.tsx

```diff
@@ -623,4 +624,26 @@ export function registerHeadlampEventCallback(callback: HeadlampEventCallback) {
   store.dispatch(addEventCallback(callback));
 }
 
+/**
+ * Register a plugin settings component.
+ * @param name - The name of the plugin.
+ * @param component - The component to use for the settings.
+ * @param displaySaveButton - Whether to display the save button.
+ *
+ * @example
+ *
+ * ```tsx
+ * import { registerPluginSettings } from '@kinvolk/headlamp-plugin/lib';
+ *
+ * registerPluginSettings('my-plugin', MyPluginSettingsComponent, true);
```

Comment:
Can you please improve this example? Looking at this, I wouldn't know how to use it.

Some ideas...
- Adding a minimal component inline (as an arrow function) if it's small, or a minimal function component.
- Seeing true here, I don't see what it means. Perhaps adding a `const displaySaveButton = true;` and then using that in the function call.